src/cli/repl/core.ts

```diff
@@ -22,7 +24,36 @@ const defaultHistoryFile = path.join(os.tmpdir(), '.flowrhistory')
  * Used by the repl to provide automatic completions for a given (partial) input line
  */
 export function replCompleter(line: string): [string[], string] {
-	return [replCompleterKeywords.filter(k => k.startsWith(line)), line]
+	const splitLine = splitAtEscapeSensitive(line)
+	// did we just type a space (and are starting a new arg right now)?
+	const startingNewArg = line.endsWith(' ')
+
+	// if we typed a command fully already, autocomplete the arguments
+	if(splitLine.length > 1 || startingNewArg){
+		const commandNameColon = replCompleterKeywords.find(k => splitLine[0] === k)
+		if(commandNameColon) {
+			const completions: string[] = []
+
+			const commandName = commandNameColon.slice(1)
+			if(getCommand(commandName)?.script === true){
+				// autocomplete script arguments
+				const options = scripts[commandName as keyof typeof scripts].options
+				completions.push(...getValidOptionsForCompletion(options, splitLine).map(o => `${o} `))
+			} else {
+				// autocomplete command arguments (specifically, autocomplete the file:// protocol)
+				completions.push('file://')
```

Comment:
maybe we can define this magic prefix as a global constant just so it is not repeated too often? :pretty-please: